vslib/SwitchStateBase.cpp

```diff
@@ -2260,7 +2270,10 @@ sai_status_t SwitchStateBase::refresh_port_oper_speed(
     }
     else
     {
-        if (!vs_get_oper_speed(port_id, attr.value.u32))
+        if (m_switchConfig->m_operSpeedIsConfiguredSpeed) {
+            attr.id = SAI_PORT_ATTR_SPEED;
+            CHECK_STATUS(get(SAI_OBJECT_TYPE_PORT, port_id, 1, &attr));
+        } else if (!vs_get_oper_speed(port_id, attr.value.u32))
```

Comment:
refactor to move { to new line and else if also to new line like rest of the code